docs/system-requirements/software/perl.rst

```diff
@@ -17,11 +17,11 @@ CPAN
 ----------------------------
 
 CPAN stands for the Comprehensive Perl Archive Network - and is the software repository for
-Perl modules. (you can compare it to http://pypi.python.org/pypi, Yum/Apt
-repositories in Linux, or even Apple's App Store). If you have Perl you have CPAN. (To check
-type `cpan` in a terminal).
+Perl modules. (you can compare it to `python's pypi <http://pypi.python.org/pypi/>`_, Yum/Apt
+repositories in Linux, or Apple's App Store). If you have Perl  installed you have CPAN. 
+To check type `cpan` in a terminal.
 
```

Comment:
I think this is better;
Perl modules (you can compare it to `python's pypi <http://pypi.python.org/pypi/>`_, Yum/Apt
repositories in Linux or Apple's App Store).